docs/json-argument-source.adoc

```diff
@@ -145,6 +145,59 @@ This parametrized test will generate the following tests:
 * [1] Snowspeeder, 4.5
 * [2] Imperial Speeder Bike, 3
 
+== JSON Parser Integration
+
+To not make users' dependency management more complex, JUnit Pioneer has no run-time dependencies.
+At the same time, it's not parsing JSON itself and relies on third-party libraries for that.
+For this extensions that means that projects who want to use it need to pull in a JSON parser themselves.
+This is the list of supported parsers (with the version used when building Pioneer although compatible versions should work as well):
+
+* Jackson: com.fasterxml.jackson.core:jackson-databind:2.13.2.2
```

Comment:
I do not think that it is needed to add the exact specific version of Jackson for this. We are not using any specific features from Jackson for this. I would only mention that Jackson is used

Reply:
I wasn't comfortable with that either, but I thought it outweighed the issue of users having to figure out the version for themselves. Because in a paragraph that tells you to add a Jackson dependency, wouldn't you expect it to tell you which version?

What if I change the sentence in 153 to:

> This is the list of supported parsers - with the version used when building Pioneer although most or even all versions should work because we're not using any of their features directly (you are) and just configure them:

Reply:
The Maven snippet uses only `$VERSION`, which means that users cannot copy paste it anyways (and I agree with the fact that they need to put a specific version). 

What if we change sentence in 153 to:

>This is the list of supported parsers:

and the line in 155 with a link to https://search.maven.org/artifact/com.fasterxml.jackson.core/jackson-databind? 

> * [Jackson](https://search.maven.org/artifact/com.fasterxml.jackson.core/jackson-databind)

Reply:
I didn't want to repeat the version to make updating it less error prone. I'm still not sure whether eliding it completely is such a good idea, but let's give it a try.